project_event/models/project_project.py

```diff
@@ -123,8 +123,18 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
+        message = '<br>'
+        message += _('Event: <br>') + self.name + '<br>'
+        for activity in self.task_ids:
+            message += _('Activity: ') + activity.name + '<br> Tasks:'
+            for index, task in enumerate(activity.child_ids):
+                message += task.name
+                if index < len(activity.child_ids)-1:
+                    message += ', '
```

Comment:
@eilst : Is the space required ', '???? .  Can we write like this 
if index < len(activity.child_ids) - 1:
  message += task.name + ','
else :
 message += task.name + '<br>'